api-docs/router-dashboard/get-neighbors.md

```diff
@@ -17,19 +17,23 @@ GET
 ### Content
 
 ```
-   [
-   	{
-   		"nickname": "fd00::2",
-   		"route_metric_to_exit": 0,
-   		"total_payments": 0,
-   		"debt": 0
-   	},
-   	{
-   		"nickname": "fd00::7",
-   		"route_metric_to_exit": 0,
-   		"total_payments": 0,
-   		"debt": 0
-   	}
+[
+   {
+      "nickname": "fd00::2",
+      "route_metric_to_exit": 0,
```

Comment:
@drozdziak1 do we have a function in babel monitor that satisfies "get route quality for route x from neigh y"